README.md

```diff
@@ -2,64 +2,58 @@
 
 Template to start from before starting frontend projects.
 
-## Install template
-
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
-
-All packages will be installed after executing the script above.
+## Task
 
-## Development mode 
-
-```bash
-$ npm run start
-```
+You need to create function that will be generate calendar for given pair 
+(year, month).
+Calendar must be table with `thead` and `tbody`, where day it's `TD`.
```

Comment:
maybe inline screenshot here?

Reply:
I think it's not necessary. If they don't know what is this, they will find it in google